src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -71,7 +92,7 @@ BasicDateInput.propTypes = {
   /**
    * BasicDateInput's value
    */
-  value: PropTypes.object.isRequired,
+  value: PropTypes.object,
```

Comment:
The prop _value_ is not required but there is no default value for it. Is it voluntary ?

Reply:
It's the same thing explained above :slightly_smiling_face: